source/includes/connection-snippets/scram-snippets/c-connection.c

```diff
@@ -0,0 +1,24 @@
+// begin scram connection
+#include <mongoc/mongoc.h>
+
+int
+main (int argc, char *argv[])
+{
+   mongoc_database_t *database;
+   mongoc_client_t *client;
+
+   mongoc_init ();
+
+   client = mongoc_client_new(
+      "mongodb+srv://<username>:<password>@<cluster-url>/test?retryWrites=true&w=majority"
```

Comment:
As previously mentioned in https://github.com/mongodb/docs-ecosystem/pull/573#discussion_r348462260, `/<db-name>` has no relevance for any drivers. It _may_ be used by some higher level libraries/ODMs, but that's outside the scope of any of these code examples.

I also assume we don't need to specify the authentication database via `authSource` since we're relying on that to be specified in the SRV/TXT result.

Furthermore, it's a bit odd that all of these connection strings were using `/test` but most of the examples select the `testDB` database. That's a bit inconsistent even if we ignore the issue of `/<db-name>` having no function.